dali/operators/image/color/brightness_contrast.cc

```diff
@@ -46,6 +36,15 @@ This operator can also change the type of data.)code")
 For signed types, 1.0 represents the maximum positive value that can be represented by
 the type.)code",
                     0.0f, true)
+    .AddOptionalArg("dtype",
+                    R"code(Output data type.
+
+If not set, the input type is used.)code", DALI_NO_TYPE);
+
+DALI_SCHEMA(Contrast)
+    .DocStr(R"code(Changes the color contrast of the image.)code")
```

Comment:
color contrast -> color (?)

Also, I'd keep the formula here, for completeness

Reply:
```suggestion
    .DocStr(R"code(Adjusts the contrast of the image.)code")
```

Reply:
done